Update Mainnet configs for WeMix and Kroma tests

jasonmci opened the pull request:
## Motivation


## Solution

jasonmci committed:
Update mainnet configs

AnieeG commented on integration-tests/ccip-tests/testconfig/override/mainnet.toml:
```diff
@@ -27,30 +27,108 @@ Data = """
+          "fee_token": "",
+          "bridge_tokens": ["0xC1F6f7622ad37C3f46cDF6F8AA0344ADE80BF450"],
+          "bridge_tokens_pools": ["0xd9547B702673c61c84bb2dF6FDf6D2F5309fE536"],
+          "arm" : "0x6E4d2dBBF8a1A943412aD451422FE11A25C781DE",
```
This should be `0xB59779d3364BC6d71168245f9ebb96469E5a5a98`

jasonmci replied:
fixing now.

jasonmci committed:
Update ARMs

jasonmci committed:
Update ARMs

jasonmci committed:
Update with bridge tokens

jasonmci committed:
Update contracts json with 1.2 configs

jasonmci committed:
Revert contracts.json add contracts-1.2.json

jasonmci committed:
Add AVAX lanes

jasonmci committed:
Update 1.2 lanes
contracts1.2.json has all configs with no receiver dapp
mainnet.toml has all configs with receiver dapp

jasonmci committed:
add receiver dapps

jasonmci committed:
Merge branch 'ccip-develop' into CCIP-1460-Smoke-test-Mainnet-Lanes

jasonmci committed:
Fix the Kroma ARM to use the correct contract
